CONTRIBUTING.md

```diff
@@ -1,81 +1,126 @@
+(e.g., see [tedlium2-example](https://github.com/espnet/espnet/blob/master/egs/tedlium2/asr1/RESULTS.md)).
    
-   To make a report for `RESULTS.md`
-	 - execute `show_result.sh` at a recipe main directory (where `run.sh` is located), as follows. 
-	   You'll get environmental information and the evaluation result of each experiments in a markdown format.
-	   ```
-	   $ show_result.sh
-	   ```
-	 - execute `pack_model.sh` at a recipe main directory as follows. You'll get model information in a markdown format
-	   ```
-	   $ pack_model.sh --lm <language model> <tr_conf> <dec_conf> <cmvn> <e2e>
-	   ```
-	 - `pack_model.sh` also produces a packed espnet model (`model.tar.gz`). If you upload this model to somewhere with a download link,
-	   please put the link information to `RESULTS.md`.
-	 - please contact Shinji Watanabe <shinjiw@ieee.org> if you want a web storage to put your model files.
-
-Once you finish implementing a feature or bugfix, please send a Pull Request to https://github.com/espnet/espnet
+To make a report for `RESULTS.md`
+ - execute `show_result.sh` at a recipe main directory (where `run.sh` is located), as follows. 
+   You'll get environmental information and the evaluation result of each experiment in a markdown format.
+   ```
+   $ show_result.sh
+   ```
+ - execute `pack_model.sh` at the main directory of a recipe as follows. You'll get model information in a markdown format
+   ```
+   $ pack_model.sh --lm <language model> --dict <dict> <tr_conf> <dec_conf> <cmvn> <e2e>
+   ```
+ - `pack_model.sh` also produces a packed espnet model (`model.tar.gz`). If you upload this model to somewhere with a download link,
+   please put the link information to `RESULTS.md`.
+ - please contact Shinji Watanabe <shinjiw@ieee.org> if you want web storage to put your model files.
+
+#### 1.3.2 ESPnet2 recipes
+
+ESPnet2's recipes correspond to `egs2`. ESPnet2 applies a new paradigm without dependencies of Kaldi's binaries, which makes it lighter and more generalized.
+For ESPnet2, we do not recommend preparing recipe's stages for each corpus but rather using the common pipelines we provided in `asr.sh`, `tts.sh`, and 
+`enh.sh`. For details of creating ESPnet2 recipes, please refer to [egs2-readme](https://github.com/espnet/espnet/blob/master/egs2/TEMPLATE/README.md).
+
+The common pipeline of ESPnet2 recipes will take care of the `RESULTS.md` generation, model packing, and uploading. ESPnet2 models are maintained at Zenodo.
+To upload your model, you need first:
+1. Sign up to Zenodo: https://zenodo.org/
+2. Create access token: https://zenodo.org/account/settings/applications/tokens/new/
+3. Set your environment: % export ACCESS_TOKEN="<your token>"
+
+#### 1.3.3 Additional Checklist when building recipes
+
+- [ ] common files are linked with symlink: we ask to use symlink to refer common scripts and utilities (e.g., `utils`, `steps`, `asr.sh`). Please use `ln -sf <source_path> <target path>` rather than a new copy.
```

Comment:
- Common/shared files and directories such as `utils`, `steps`, `asr.sh`, etc, should be linked using a symbolic link (e.g.: `ln -s <source-path> <target-path>`). Please refer to existing recipes if you're unaware which files/directories are shared.